crates/aptos-protos/proto/aptos/util/timestamp/timestamp.proto

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.util.timestamp;
+
+message Timestamp {
+  // Represents seconds of UTC time since Unix epoch
+  // 1970-01-01T00:00:00Z. Must be from 0001-01-01T00:00:00Z to
+  // 9999-12-31T23:59:59Z inclusive.
+  int64 seconds = 1;
+
+  // Non-negative fractions of a second at nanosecond resolution. Negative
+  // second values with fractions must still have non-negative nanos values
+  // that count forward in time. Must be from 0 to 999,999,999
+  // inclusive.
+  int32 nanos = 2;
```

Comment:
Non negative, let's use an unsigned type then?

Reply:
I think this comes from here [link](https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/timestamp.proto). let's keep this sync with open-sourced one.